Content.Server/StationEvents/Events/CluwneBeastSpawn.cs

```diff
@@ -0,0 +1,30 @@
+using System.Linq;
+using Content.Server.StationEvents.Components;
+
+namespace Content.Server.StationEvents.Events;
+
+public sealed class CluwneBeastSpawn : StationEventSystem
+{
+    public override string Prototype => "CluwneBeastSpawn";
+
+    public override void Started()
+    {
+        base.Started();
+        var spawnLocations = EntityManager.EntityQuery<VentCritterSpawnLocationComponent>().ToList();
+        RobustRandom.Shuffle(spawnLocations);
+
+        var mod = Math.Sqrt(GetSeverityModifier());
+
+        var spawnAmount = (1);
```

Comment:
why ()

Reply:
Not exactly sure how to setup a config, i searched for it but failed to find any real examples. Could you please expalin it a little?

Reply:
check out how ninja does it its basically the exact same with config in gameticking/rules/configuration and the event in stationevents/events

Reply:
thanks, this should be all sorted now.